### PR TITLE
Fix typo in code splitting code sample

### PR DESCRIPTION
**Pull request by polotek**
Minor content fix for correctness. The previous code example ran `someFunction` immediately instead of inside the promise callback.

### CODE REVIEW

**Comment by heyawhite** [comment]
Adding @housseindjirdeh who wrote the article and @malchata for performance expertise -- if one of you confirms, I'll merge

**Comment by malchata** [comment]
Thanks for catching this @polotek! All looks good to me. Only thing is we'll need you to [sign the CLA](https://opensource.google/documentation/reference/cla) before the workflow will allow us to merge your contribution.

**Review by housseindjirdeh — APPROVED** [review]

**Comment by miguelfdezc** [comment]
Hi @polotek! Are you still working on this fix?

If not, I'm willing to take over and create an issue and PR to fix the code so we can improve the documentation

@heyawhite @housseindjirdeh @malchata waiting for your response to make my contribution

**Review by malchata — APPROVED** [review]

**Comment by malchata** [comment]
@miguelfdezc did you sign the CLA? If not, please do so we can merge this.

**Comment by miguelfdezc** [comment]
Hi @malchata, I did sign the CLA. I believe you wanted to tag @polotek instead?
Actually, the reason why I commented was that I wanted to fix this too but I was there was this PR already but it was stale because of the missing CLA sign

**Comment by malchata** [comment]
@polotek, if you can sign the CLA, I can merge this change. Sorry for the hassle!

**Comment by matthiasrohmer** [comment]
@miguelfdezc, feel free to go ahead and open another PR with the fix while mentioning this one 🙂

**Comment by matthiasrohmer** [comment]
Superseded by #8589.

**Comment by polotek** [comment]
Thank you for trying to get in touch to give me full attribution. I have now signed the CLA for future reference. But I'm fine with how this ticket was resolved.

**Pull request closed without merging**
